packages/cli/src/commands/site/index.ts

```diff
@@ -61,32 +61,24 @@ const command: CommandModule<CliCommonArgs, CommonSiteArgs> = {
           description: "Path to a file containing your Foundry API Token",
         },
       })
+      .coerce({
+        foundryUrl: (foundryUrl) => foundryUrl.replace(/\/$/, ""),
+      })
```

Comment:
Can we make this match the existing coerce where we do it directly in the option definition it relates to? That way I think we don't need to specify the key again and it's more tightly grouped with the definition